app/views/play/CampaignView.coffee

```diff
@@ -622,7 +623,14 @@ module.exports = class CampaignView extends RootView
       level.locked = true if level.requiresSubscription and @requiresSubscription and me.get('hourOfCode')
       level.locked = false if @levelStatusMap[level.slug] in ['started', 'complete']
       level.locked = false if @editorMode
-      level.locked = false if not @course? and @campaign?.get('name') in ['Auditions', 'Intro']
+      if @jqxhr
```

Comment:
This if makes these section course-only, but the `if not @course? .. ` was specifically meant to be run in non-course scenarios.